chsdi/templates/htmlpopup/hiks_dufour_metadata.mako

```diff
@@ -0,0 +1,7 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <tr><td class="cell-left">${_('ch.swisstopo.hiks-dufour.metadata.s_map_number')}</td>         <td>${c['attributes']['s_map_number'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.swisstopo.hiks-dufour.metadata.s_title_de')}</td>           <td>${c['attributes']['s_title_de'] or '-'}</td></tr>
```

Comment:
No there are titles for all the languages. However, the title is the same for all of them. For this reason i only used s_title_de. So, do i have to add s_title_fr, s_title_it, etc for all the languages even if they are the same?